### PR TITLE
Style/new brand

### PR DESCRIPTION
**Pull request by pedroferreira1**
image
image
image
image
image
image

### CODE REVIEW

File: src/components/TokenSelect.js [hunk 1]
```diff
@@ -33,7 +34,7 @@ const TokenSelect = (props) => {
       <TouchableHighlight
         style={index === 0 ? styles.firstItemWrapper : null}
         onPress={() => { props.onItemPress(item); }}
-        underlayColor='rgba(227, 0, 82, 0.5)'
+        underlayColor='rgba(140, 70, 255, 0.3)'
```
Comment (msbrogli) [inline src/components/TokenSelect.js]: Isn't it better to include as a constant as well?

File: src/screens/MainScreen.js [hunk 2]
```diff
@@ -408,13 +409,13 @@ class BalanceView extends React.Component {
       marginBottom: 24,
     },
     networkView: {
-      backgroundColor: 'rgba(227, 0, 82, 0.1)',
+      backgroundColor: 'rgba(140, 70, 255, 0.1)',
```
Comment (msbrogli) [inline src/screens/MainScreen.js]: Isn't it better to include it as a constant? It seems to be the same as in TokenSelect.js, isn't it?

File: src/screens/Settings.js [hunk 3]
```diff
@@ -48,26 +50,26 @@ export class Settings extends React.Component {
       marginBottom: 24,
     },
     networkView: {
-      backgroundColor: 'rgba(227, 0, 82, 0.1)',
+      backgroundColor: 'rgba(140, 70, 255, 0.1)',
```
Comment (msbrogli) [inline src/screens/Settings.js]: Again, the same color here.

File: src/components/TokenSelect.js (hunk 1 above)
Comment (pedroferreira1) [reply src/components/TokenSelect.js]: Created a method in constants to get this color depending on the alpha value.

File: src/screens/Settings.js (hunk 3 above)
Comment (pedroferreira1) [reply src/screens/Settings.js]: Created a method in constants to get this color depending on the alpha value.

File: src/screens/MainScreen.js (hunk 2 above)
Comment (pedroferreira1) [reply src/screens/MainScreen.js]: Created a method in constants to get this color depending on the alpha value.

**Pull request merged**